src/IntegrationTests/BuiltInTypes/ByteArray.cs

```diff
@@ -1,75 +1,76 @@
 ﻿using System.ComponentModel.DataAnnotations;
-using System.Data.Entity;
 using System.Linq;
+using System.Threading.Tasks;
+using AutoMapper.UnitTests;
+using Microsoft.EntityFrameworkCore;
 using Shouldly;
 using Xunit;
 
-namespace AutoMapper.IntegrationTests
+namespace AutoMapper.IntegrationTests.BuiltInTypes;
+
+public class ByteArrayColumns : AutoMapperSpecBase, IAsyncLifetime
```

Comment:
Maybe have a base class `IntegrationTest<TDbInitializer> : IAsyncLifetime`?

Reply:
Yeah that can work. Though I'm not sure I want to have even more inheritance in our tests classes. The whole "Establish_Context" "Because_of" should probably all go away.

Reply:
Yeah, those should go  :)

Reply:
In order to target the linux build servers, I'll have to also move to SQlite from LocalDB, so I was gonna wait for any additional refactorings to see what that looks like.

Reply:
Some SQL things don't work on SQLite, so I don't know about that, probably just run what does work.